serverless/pages/ml-nlp-auto-scale.mdx

```diff
@@ -0,0 +1,131 @@
+---
+slug: /serverless/general/ml-nlp-auto-scale
+title: Trained model autoscaling
+tags: ['serverless']
+---
+
+You can enable autoscaling for each of your trained model deployments.
+Autoscaling allows Elasticsearch to automatically adjust the resources the model deployment can use based on the workload demand.
+
+There are two ways to enable autoscaling:
+
+- through APIs by enabling adaptive allocations
+- in Kibana by enabling adaptive resources
+
+<DocCallOut color="primary" title="Note">
+  To fully leverage model autoscaling, it is highly recommended to enable [Elasticsearch deployment autoscaling](https://www.elastic.co/guide/en/cloud/current/ec-autoscaling.html).
+</DocCallOut>
+
+Trained model autoscaling is available for Search, Observability, and Security projects on serverless deployments. However, these projects handle processing power differently, which impacts their costs and resource limits.
```

Comment:
❓  do we call it autoscaling or adaptive allocations?

Reply:
As far as I understand, trained model autoscaling is an umbrella term to cover both methods: adaptive allocations which is the API way with specific settings (allocations and threads), and adaptive resources which is the UI way and primarily uses the `vCU` / `vCPU` terminology. This is why we say in the beginning of the page that:

```
There are two ways to enable autoscaling:

- through APIs by enabling adaptive allocations
- in Kibana by enabling adaptive resources
```

Does that make sense, @darnautov?

Reply:
Makes sense, just wanted to point that out, as we also have the ML autoscaling term in the Cloud Console. To avoid any confusion